src/main/java/training/endava/skeleton/Main.java

```diff
@@ -153,13 +157,111 @@ public int compare(Article o1, Article o2) {
 
         }
 
+*/
 
+        List<Company> compList = new ArrayList<>();
+        List<Person> persList = new ArrayList<>();
+        Company comp1 = new Company("Company1", 1993, persList, CompanyArea.IT);
+        Person pers1 = new Person(1, "Hirtan Daniel", "555-000-111", "Strada Arhitect Ioan nr. 16", comp1);
+        persList.add(pers1);
+        Person pers2 = new Person(2, "Alex ABC", "555-555-555", "Strada Pacurari nr. 123456", comp1);
+        persList.add(pers2);
+        Person pers3 = new Person(3, "Victor ABC", "555-585-888", "Strada Tatarasu nr. 12", comp1);
+        persList.add(pers3);
 
 
-    }
+        compList.add(comp1);
+
+        List<Person> persList2 = new ArrayList<>();
+        Company comp2 = new Company("Company2", 1990, persList2, CompanyArea.SUPPORT);
+
+        Person pers4 = new Person(4, "ABC", "555-000888", "Strada nr. 16", comp2);
+        persList.add(pers4);
+        Person pers5 = new Person(5, " ABCDE", "555-999-555", "Strada Canta nr 123", comp2);
+        persList.add(pers5);
+        Person pers6 = new Person(6, "Victor ABC", "111-585-988", "Strada Colentina nr. 12", comp2);
+        persList.add(pers6);
+
+        compList.add(comp2);
+
+        List<String> compNameList = compList.stream().
+                map(Company::getName).collect(Collectors.toList());
+        System.out.println(compNameList);
+
+        int[] compSort = compList.stream().
+                map(Company::getYear).
+                mapToInt(Integer::intValue).sorted().toArray();
+
+        for (int i = 0; i < compSort.length; i++)
+            System.out.println(compSort[i]);
```

Comment:
Why do you use external iteration when you can replace `.toArray()` with `.forEach(System.out::println)`?